src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -65,7 +65,7 @@
           type: string
           short-summary: >
             The name of the operating system image as a URN alias, URN, custom image name or ID, or VHD blob URI.
-            This parameter is required unless using `--attach-os-disk.`
+            This parameter is required unless using `--attach-os-disk.` Valid URN format: "Publisher:Offer:Sku:Version".
```

Comment:
i can elaborate, but the image can be a managed custom image resource id or name

Reply:
The help text still says that the image could be a custom image name or ID, I can modify the help text to say **managed** custom image name or **resource** ID.